api/Dockerfile

```diff
@@ -4,7 +4,7 @@
 ## SPDX-License-Identifier: MIT
 ##
 
-ARG BUILD_BASE_IMAGE=golang:1.22.3-alpine
+ARG BUILD_BASE_IMAGE=golang:1.22.4-alpine
```

Comment:
We need corresponding change in ADO repo.

Reply:
This part will be done when adding commits to ADO repo.